### ISSUE TITLE
Can we get support for IPython

### ISSUE THREAD

**Issue by thestick613** [ionelmc/python-manhole#40]
Seems much better suited for the job.

**Comment by ionelmc** [comment]
Yes, it would be nice indeed. Unfortunately I'd have to implement terminal emulation and that's not exactly easy and I'm trying to keep the dependencies minimal and the library simple.

Which features of IPython would be useful for you? Autocompletion? History? Line editing? IPython "Magics" (the `%` commands)`?

**Comment by thestick613** [comment]
Mostly autocompletion and history.

**Comment by ionelmc** [comment]
For history (and line editing) you can use `manhole-cli` or `socat` (as described in the readme).
